plume-front/src/main.rs

```diff
@@ -19,7 +20,33 @@ fn auto_expand() {
                 js! {
                     @{&article_content}.style.height = "auto";
                     @{&article_content}.style.height = @{&article_content}.scrollHeight - @{offset} + "px";
-                }
+                };
+                window().set_timeout(|| {match document().query_selector("#post-form") {
+                    Ok(Some(form)) => HtmlElement::try_from(form).map(|form| {
+                        if let Some(len) = form.get_attribute("content-size").and_then(|s| s.parse::<i32>().ok()) {
+                            let consumed: i32 = js!{
```

Comment:
whoa! `js!`?!

Reply:
Yes, it is macro to call JS code in WASM, when you are too limited by Rust APIs (since they are still quite experimental).

Reply:
the rational here is that stdweb doesn't support [HTMLFormElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement) yet, and querying recursively every child to test if it's an input is annoying. And to not pass variables back and fourth a lot of time, I made most computations in js, those are only a few add anyway.
If you want to read it, things in `@{}` are rust vars (and possibly code I think), other things are plain js